### DISCUSSION TITLE
启动导入的整合包，在游戏界面有关打开文件夹的地方都找不到路径

### DISCUSSION THREAD

**Discussion by ChestnutPlumage** [Meloong-Git/PCL#935] [提问] [answered]
如图所示(SkyFactory4-4.2.4整合包）
![image](https://user-images.githubusercontent.com/122587720/212632725-6fc184b7-4043-40fd-85f9-75b4da13c977.png)
![image](https://user-images.githubusercontent.com/122587720/212632782-70b7badf-f06c-44b2-a30c-78660a72671f.png)
但是确确实实我是没有改过路径，并且都是英文路径的，如下图
![image](https://user-images.githubusercontent.com/122587720/212633136-fe08d118-8a98-446e-a31e-f992e1730459.png)
不过问题只是出在这个整合包，请问怎样才可以在游戏界面正常打开对应文件夹？

**Comment by Pigeon0v0** [comment]
看起来你曾经把某个用户目录设置在了 D 盘根目录...

我这里最近有相似经历，我是通过重置系统解决的

但是你说问题只出在这个整合包也挺奇怪的

**Reply by ChestnutPlumage** [reply]
对，才开始用电脑的时候换过d盘，但是发现c盘还是有东西，所以又还回去了（当时还不太了解）
重置系统是指电脑设置的初始化那个吗，还是说要重装系统，我只初始化过，初始化以后就没有做过这些了
只是这个整合包，其他的我都可以打开，不管是在pcl2里面下载那下载的还是锻造诅咒网站下载导入的

**Reply by Pigeon0v0** [reply]
> 对，才开始用电脑的时候换过d盘，但是发现c盘还是有东西，所以又还回去了（当时还不太了解）

打开系统设置 - 系统 - 恢复 - 初始化电脑

**Reply by Pigeon0v0** [reply]
建议使用云下载重置系统

**Reply by ChestnutPlumage** [reply]
好的謝謝，我保存一下文件就試試云下載，以前只是本地下載過

**Reply by ChestnutPlumage** [reply]
没用，还是一样找不到路径，按照你的方法做了，初始化云下载的

**Reply by Pigeon0v0** [reply]
> 没用，还是一样找不到路径，按照你的方法做了，初始化云下载的
> 
> 

那我就不知道了，我自己重置了就没问题了

**Comment by WorldHim** [comment]
似乎是路径前面多了一个 `\` 导致 资源管理器 打不开路径，不知道什么问题。

**Reply by ChestnutPlumage** [reply]
但是文件夹名字不可能包含符号的，就不可能说\这种了

**Comment by LTCatt** [comment, marked as answer]
Minecraft 的 Bug

**Reply by ChestnutPlumage** [reply]
了解，谢谢，最近下载一个叫丰收空岛的整合包也是如此/苦笑
